src/config.ts

```diff
@@ -57,9 +57,9 @@ export class Config {
   };
   abi: { [contract: string]: string[] };
   seed: {
-    api: { host?: string; port: number };
-    git: { host?: string; port: number };
-    link: { host?: string; id?: string; port: number };
+    api: { host?: string | null; port: number };
+    git: { host?: string | null; port: number };
+    link: { host?: string | null; id?: string | null; port: number };
```

Comment:
How come?

Reply:
Can you point to the code that passes null?

Reply:
> Can you point to the code that passes null?

in `base/seed/View.svelte:L11` we have 
`const seed = new Seed(config, null, seedAddress);`
Since we are not passing a seed Id and we don't want to pass explicit `undefined` I had to start adding nulls.
If we don't want to do this, we can:
1. Pass `undefined` as argument for the seed id
2. Pass the seed id as query param eg. `/seed/hyyqpngdoe4x4oto3emfdppbw7sj1pfaghbpmmhz5rqiuqg8uofmeo@seed.alt-clients.radicle.xyz`

Reply:
> Can you point to the code that passes null?

in `base/seed/View.svelte:L11` we have 
`const seed = new Seed(config, null, seedAddress);`
Since we are not passing a seed Id and we don't want to pass explicit `undefined` I had to start adding nulls.
If we don't want to do this, we can:
1. Pass `undefined` as argument for the seed id
2. Pass the seed id as query param eg. `/seed/hyyqpngdoe4x4oto3emfdppbw7sj1pfaghbpmmhz5rqiuqg8uofmeo@seed.alt-clients.radicle.xyz`

Reply:
Aha, I think solution 3. is probably the cleanest for now. We may run into this again but we can deal with it then if so. Just don't forget all the other call sites for `new Seed`

Reply:
Okay will do